internal/cli/atlas/performanceadvisor/suggestedindexes/list.go

```diff
@@ -57,23 +57,26 @@ func (opts *ListOpts) Run() error {
 	if err != nil {
 		return err
 	}
-	r, err := opts.store.PerformanceAdvisorIndexes(opts.ConfigProjectID(), host, opts.newSuggestedIndexOptions())
+	r, err := opts.store.PerformanceAdvisorIndexes(opts.newSuggestedIndexOptions(opts.ConfigProjectID(), host))
 	if err != nil {
 		return err
 	}
 
 	return opts.Print(r)
 }
 
-func (opts *ListOpts) newSuggestedIndexOptions() *atlas.SuggestedIndexOptions {
-	return &atlas.SuggestedIndexOptions{
-		Namespaces: opts.namespaces,
-		NIndexes:   opts.nIndexes,
-		NExamples:  opts.nExamples,
-		NamespaceOptions: atlas.NamespaceOptions{
-			Since:    opts.since,
-			Duration: opts.duration,
-		},
+func (opts *ListOpts) newSuggestedIndexOptions(project, host string) *atlas.ListSuggestedIndexesApiParams {
+	// TODO log issue with the API
```

Comment:
Still required?

Reply:
Good catch. Done already (forgot to include jira).